doc/reference/internals/iproto/replication.rst

```diff
@@ -42,20 +42,27 @@ General
             -   0x29
             -   Response to IPROTO_VOTE. Used during replica set bootstrap
 
-        *   -   IPROTO_FETCH_SNAPSHOT
+        *   -   :ref:`IPROTO_FETCH_SNAPSHOT <box_protocol-fetch-snapshot>`
             -   0x45
             -   Fetch the master's snapshot and start anonymous replication.
-                See :ref:`replication.anon <configuration_reference_replication_anon>`
 
-        *   -   IPROTO_REGISTER
+        *   -   :ref:`IPROTO_REGISTER <box_protocol-register>`
             -   0x46
             -   Register an anonymous replica so it is not anonymous anymore
+
+        *   -   :ref:`IPROTO_JOIN_META <box_protocol-join-meta>`
+            -   0x47
+            -   A request sent in response to IPROTO_JOIN before the instance initialization information
```

Comment:
It's sent in response to IPROTO_JOIN or IPROTO_FETCH_SNAPSHOT
Same about IPROTO_JOIN_SNAPSHOT

Reply:
Thanks, added